build/cdc_data/Dockerfile

```diff
@@ -14,69 +14,71 @@
 
 
 # #### Stage 1: Download base dc model from GCS. ####
-FROM google/cloud-sdk:slim as model-downloader
+FROM google/cloud-sdk:slim AS model-downloader
 
 # Copy model.
 RUN mkdir -p /tmp/datcom-nl-models \
     && gsutil -m cp -R gs://datcom-nl-models/ft_final_v20230717230459.all-MiniLM-L6-v2/ /tmp/datcom-nl-models/
 
 
-# #### Stage 2: Copy required files. ####
-FROM python:3.11.4-slim as file-copier
+# #### Stage 2: Install python dependencies. ####
+FROM python:3.11.4-slim AS file-copier
```

Comment:
Consider renaming the stage to `dependencies-installer` or something similar.

Reply:
Done!